x-pack/plugins/ingest_pipelines/public/application/components/pipeline_editor/components/processor_form/processors/user_agent.tsx

```diff
@@ -47,6 +47,34 @@ const fieldsConfig: FieldsConfig = {
       }
     ),
   },
+  extract_device_type: {
+    type: FIELD_TYPES.TOGGLE,
+    defaultValue: false,
+    deserializer: to.booleanOrUndef,
+    serializer: from.undefinedIfValue(false),
+    label: (
+      <EuiFlexGroup gutterSize="xs">
+        <EuiFlexItem grow={false}>
+          <FormattedMessage
+            id="xpack.ingestPipelines.pipelineEditor.userAgentForm.extractDeviceNameFieldText"
+            defaultMessage="Extract Device Type"
+          />
+        </EuiFlexItem>
+        <EuiFlexItem style={{ marginBottom: -4 }}>
```

Comment:
This one felt strange, but seemed to me like an acceptable trade off while we wait until the more condensed BetaBadge is available for us to use. If we dont apply the negative margin we end up with a ton of space between the `helpText` and the field it self, making it look different from the other fields:

With negative margin:
![Screenshot 2021-05-31 at 11 46 53](https://user-images.githubusercontent.com/1191206/120174927-f2427a00-c205-11eb-9b75-08cab0f216d6.png)


Without:
![image](https://user-images.githubusercontent.com/1191206/120174957-f9698800-c205-11eb-9948-16610f95a462.png)